nodes_template/default.toml

```diff
@@ -44,4 +44,4 @@ trace = 100
 txs_verify_cache_size = 100000
 
 [block_assembler]
-type_hash = "0x0da2fe99fe549e082d4ed483c2e968a89ea8d11aabf5d79e5cbf06522de6e674"
+type_hash = "0x8954a4ac5e5c33eb7aa8bb91e0a000179708157729859bd8cf7e2278e1e12980"
```

Comment:
is it a type hash for `always_success` script?

Reply:
yes

Reply:
Please change this type hash also, and add a comment in default.toml:
https://github.com/nervosnetwork/ckb/blob/develop/test/fixtures/nodes_template/default.toml#L47